en/docs/administer/customize-the-developer-portal.md

```diff
@@ -10,30 +10,39 @@ To customize the Developer Portal theme, follow the steps given below:
 
 1. Sign in to the Choreo Console at [https://console.choreo.dev/](https://console.choreo.dev/) using a Google, GitHub, or Microsoft account.
 
-2. If you are a new user, create an organization. Enter a unique organization name. For example, "Stark Industries".
 
-    ![Create an organization in Choreo](../assets/img/references/enterprise-login/create-choreo-organization.png){.cInlineImage-small}
-
-3. In the left pane, click **Settings**.
+2. In the left pane, click **Settings**.
 
-4. In the **Organization** tab, click **Devportal Theme**.
+3. In the **API Management** tab, click **Devportal Theme**.
 
-    ![Access Devportal theme](../assets/img/administration/devportal-theme/access-devportal-theme.png)
+    ![Access Devportal theme](../assets/img/administration/devportal-theme/access-devportal-theme.png){.cInlineImage-half}
```

Comment:
{.cInlineImage-half}
@RukshiW is this correct?

Reply:
Yes. That is the image formatting applied.